kafka-server/src/test/java/com/ozangunalp/kafka/server/BrokerConfigTest.java

```diff
@@ -8,168 +8,167 @@
 
 import org.junit.jupiter.api.Test;
 
-import kafka.server.KafkaConfig;
 
 class BrokerConfigTest {
 
     @Test
     void testEmptyOverride() {
         Properties properties = BrokerConfig.defaultCoreConfig(new Properties(), "", 9092, 9093, 9094, PLAINTEXT);
-        assertThat(properties).containsEntry(KafkaConfig.BrokerIdProp(), "1");
-        assertThat(properties).containsEntry(KafkaConfig.QuorumVotersProp(), "1@:9094");
-        assertThat(properties).containsEntry(KafkaConfig.ListenersProp(), "BROKER://:9093,PLAINTEXT://:9092,CONTROLLER://:9094");
-        assertThat(properties).containsEntry(KafkaConfig.ProcessRolesProp(), "broker,controller");
-        assertThat(properties).containsEntry(KafkaConfig.ControllerListenerNamesProp(), "CONTROLLER");
-        assertThat(properties).containsEntry(KafkaConfig.InterBrokerListenerNameProp(), "BROKER");
-        assertThat(properties).containsEntry(KafkaConfig.AdvertisedListenersProp(), "PLAINTEXT://:9092,BROKER://:9093");
-        assertThat(properties).containsEntry(KafkaConfig.EarlyStartListenersProp(), "BROKER,CONTROLLER");
-        assertThat(properties).containsEntry(KafkaConfig.ListenerSecurityProtocolMapProp(), "BROKER:PLAINTEXT,CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT");
+        assertThat(properties).containsEntry("broker.id", "1");
```

Comment:
With constants now coming from so many different places, a mistake would be easy.  Testing against the string gives us confidence that the production code is achieving the right config.